blosc/fastcopy.c

```diff
@@ -99,18 +85,13 @@ static inline unsigned char *copy_16_bytes(unsigned char *out, const unsigned ch
   chunk = _mm_loadu_si128((__m128i*)from);
   _mm_storeu_si128((__m128i*)out, chunk);
   out += 16;
-#elif !defined(BLOSC_STRICT_ALIGN)
-  *(uint64_t*)out = *(uint64_t*)from;
-   from += 8; out += 8;
-   *(uint64_t*)out = *(uint64_t*)from;
-   from += 8; out += 8;
 #else
-   int i;
-   for (i = 0; i < 16; i++) {
-     *out++ = *from++;
-   }
-#endif
+  int i;
+  for (i = 0; i < 16; i++) {
+    *out++ = *from++;
+  }
   return out;
```

Comment:
This is just a `memcpy()` as well. Given that compilers may already replace that loop with a call to memcpy() anyway just use that.